src/Api/Email.php

```diff
@@ -44,6 +44,33 @@ public function __construct(UnioneClient $client)
    */
   public function send(array $params, array $headers = []): array
   {
+      $headers_to_normalise = ['to', 'cc', 'bcc'];
+      foreach ($headers as $key => $header) {
+        $test = strtolower($key);
+        if (in_array($test, $headers_to_normalise)) {
+          if ($test !== $key) {
+            $headers[$test] = $headers[$key];
+            unset($headers[$key]);
+          }
+        }
+      }
+
+      $recipients = [];
+      // Prepare to remove duplicates if any.
+      if (is_array($params['recipients'])) {
```

Comment:
\is_array

Reply:
Use fully qualified functions (FQN) for all added code.